docs/radio.rst

```diff
@@ -132,6 +133,26 @@ Functions
 
     A ``ValueError`` exception is raised if conversion to string fails.
 
+.. py:function:: receive_full()
+
+    Returns a tuple containing three values representing the next incoming
+    message on the message queue. If there are no pending messages then
+    ``None`` is returned.
+
+    The three values in the tuple represent:
+
+    * the next incoming message on the message queue as bytes.
+    * the RSSI (signal strength): a value between 0 (strongest) and -255 (weakest) as measured in dBm.
+    * a timestamp: the value returned by `microbit.current_time` when the message was receieved.
+
+    For example::
+
+        msg, rssi, timestamp = radio.receive_full()
```

Comment:
This is not a good example if the method is going to return None when no messages are available.  Either receive_full should return a 3-tuple of None's indicating no message, or this example should assign to just a single variable.